src/main/java/org/jabref/logic/bibtex/FieldContentFormatter.java

```diff
@@ -47,7 +47,7 @@ public String format(String fieldContent, Field field) {
             return fieldContent;
         }
 
-        return WHITESPACE.matcher(fieldContent).replaceAll(" ");
+        return WHITESPACE.matcher(fieldContent).replaceAll(" ").replaceAll("&amp;#x2014;", "—");
```

Comment:
This is not the right place as  this can have unforseen consequences

better create a new formatter, see for example NormalizeEnDashesFormatter

Reply:
Okay, if I create new formatter how to add it to the parser? Or just where to add it, so it will format strings?